yumi_description/launch/yumi.rviz

```diff
@@ -1,236 +0,0 @@
-Panels:
```

Comment:
why did you delete the rviz configuration, I think in general it's nice to have one.